### ISSUE TITLE
Tracking issue: ICU-21545  Export UCPTrie data to array buffers

### ISSUE THREAD

**Issue by sffc** [unicode-org/icu4x#509]
***This is the ICU4X tracking issue for https://unicode-org.atlassian.net/browse/ICU-21545***

ICU4X needs code point trie data (both tests and for properties) exported to an easily consumable format for downstream use.

Related: #508, #132

**Comment by sffc** [comment]
Once https://github.com/unicode-org/icu/pull/1857 is merged, I will consider this task complete.  We will need a follow-up issue to automatically ingest the ICU data files once they are actually shipped in ICU 70.

**Comment by sffc** [comment]
This is complete for binary and enumerated properties.  We don't yet handle Script_Extensions; I followed https://github.com/unicode-org/icu4x/issues/1086 to follow up on that.
